guides/release/configuring-ember/optional-features.md

```diff
@@ -196,10 +196,10 @@ can be set to run asynchronously using the `sync: false` option.
 import { observer } from '@ember/object';
 
 Person.extend({
```

Comment:
maybe `Image`, or `Screen`, to go better with the aspect ratio theme

Reply:
Yeah, I think the name can be changed. I'm not sure which of the two better illustrates a synchronous observer.

Reply:
Addressed in 39a216a8